Marlin/dogm_lcd_implementation.h

```diff
@@ -17,8 +17,8 @@
 #define MARLIN_VERSION "1.0.2"
```

Comment:
I suppose we _could_ use `STRING_VERSION` from `Configuration.h` instead of having to change this when Marlin changes.